clang/test/Driver/sycl-MD-default.cpp

```diff
@@ -6,6 +6,7 @@
 // RUN:   | FileCheck -check-prefix=CHK-DEFAULT %s
 // RUN: %clang_cl -### -MDd -fsycl -c %s 2>&1 \
 // RUN:   | FileCheck -check-prefix=CHK-DEFAULT %s
+// CHK-DEFAULT-NOT: "-fsycl-is-device" {{.*}} "-D_MT" "-D_DLL"
 // CHK-DEFAULT: "-D_MT" "-D_DLL"
```

Comment:
For clarity & better coherence with the new check:
```suggestion
// CHK-DEFAULT: "-fsycl-is-host" {{.*}}  "-D_MT" "-D_DLL"
```